test-runner/src/main/scala/scala/scalanative/testinterface/ComRunner.scala

```diff
@@ -56,51 +55,44 @@ class ComRunner(bin: File,
     new BufferedOutputStream(socket.getOutputStream))
 
   /** Send message `msg` to the distant program. */
-  def send(msg: Message): Unit = synchronized {
-    try SerializedOutputStream(out)(_.writeMessage(msg))
-    catch {
-      case ex: Throwable =>
-        close()
-        throw ex
+  def send(msg: Message): Unit = {
+    synchronized { // Here, not at def, to workaround SN Issue #1091.
```

Comment:
This code runs on the JVM, compiled by Scala/JVM. There's no need to work around a Native codegen issue.

Reply:
Thank you for timely and detailed review, here and throughout this PR.  
I appreciate the nits now being gone!  The quality has certainly improved
through the PR process.